tests/test_realtime_web.py

```
@@ -43,15 +58,32 @@ def test_realtime_assets_are_authenticated(tmp_path, monkeypatch):
     assert ".filter(matches)" in script.text
     assert "const start = timing(left).end" in script.text
     assert '<h3>Timing ›</h3>' in script.text
+    assert '<h3>Fields ›</h3>' in script.text
```

Ruff (0.16.1)

[warning] String contains ambiguous `›` (SINGLE RIGHT-POINTING ANGLE QUOTATION MARK). Did you mean `>` (GREATER-THAN SIGN)?

(RUF001)